tracee-ebpf/tracee/consts.go

```diff
@@ -968,4 +972,6 @@ var EventsIDToParams = map[int32][]external.ArgMeta{
 	SecurityInodeMknodEventID:     {{Type: "const char*", Name: "file_name"}, {Type: "umode_t", Name: "mode"}, {Type: "dev_t", Name: "dev"}},
 	InitNamespacesEventID:         {{Type: "u32", Name: "cgroup"}, {Type: "u32", Name: "ipc"}, {Type: "u32", Name: "mnt"}, {Type: "u32", Name: "net"}, {Type: "u32", Name: "pid"}, {Type: "u32", Name: "pid_for_children"}, {Type: "u32", Name: "time"}, {Type: "u32", Name: "time_for_children"}, {Type: "u32", Name: "user"}, {Type: "u32", Name: "uts"}},
 	SocketDupEventID:              {{Type: "int", Name: "oldfd"}, {Type: "int", Name: "newfd"}, {Type: "struct sockaddr*", Name: "remote_addr"}},
+	HookedFopsPointerEventID:	   {{Type: "u64", Name: "/proc_fops_hooked_by"},{Type: "u64", Name: "/proc_iterate_shared_function_hooked_by"}},
```

Comment:
nit: unaligned

Reply:
nit: missing space after comma

Reply:
are you sure /proc_fops_hooked_by is a good name ? `procfs_fops_hooked` instead ?